app/src/main/java/com/team3663/scouting_app/activities/PreMatch.java

```diff
@@ -336,6 +340,9 @@ private void initNext() {
                     }
 
                     // Save off some fields for next time or later usage
+                    if ((ScouterName != null) && (!ScouterName.isEmpty()) && !ScouterName.equals(String.valueOf(preMatchBinding.editScouterName.getText())))
+                        Globals.myAchievements.clearAllData();
```

Comment:
Ditto

Reply:
Ditto.  No initAchievments in pre-match.   If there's a new scouter, we need to clear out the achievement data.